client/cypress/e2e/downloadPacket.spec.js

```diff
@@ -2,22 +2,21 @@
 
 describe('Census Block Group download', () => {
   it('validate file download', () => {
-    // const filename = `usa.zip`;
-    cy.visit('localhost:8000/en/cejst');
+    const filename = `Screening+Tool+Data.zip`;
```

Comment:
`+` in some filesystems can cause issues. Maybe convert them to something else?
see: https://www.mtu.edu/umc/services/websites/writing/characters-avoid/

Reply:
Hi @esfoobar-usds ,

How hard would it be to change filename to remove whitespaces?

Reply:
Very easy

Reply:
@esfoobar-usds Great!

Should we do that now on this ticket or create a separate ticket?